### ISSUE TITLE
[REFACTOR]: [Globalpay] MCA metadata validation

### ISSUE THREAD

**Issue by swangi-kumari** [juspay/hyperswitch#2911]
### :memo: Feature Description

- We don't have validation for MCA metadata, So while creating MCA we are allowing any json value. Because of which RequestEncodingFailure happening during payments for the MCA which doesn't have the expected fields

### :hammer: Possible Implementation

- While adding Merchant Connector Account we want that metadata field should be validated. 
- You can check this PR for further reference https://github.com/juspay/hyperswitch/pull/2746
:bookmark: Note: All the changes needed should be contained within `hyperswitch/crates/router/src/connector/globalpay` and `crates/router/src/core/admin.rs`

### :package: Have you spent some time checking if this feature request has been raised before?

- [X] I checked and didn't find a similar issue

### :package: Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/juspay/hyperswitch/blob/main/docs/CONTRIBUTING.md)

### :sparkles: Are you willing to submit a PR?

**Comment by smeet07** [comment]
@swangi-kumari can you assign this issue to me?

**Comment by swangi-kumari** [comment]
Sure @smeet07 ,
I have assigned this issue to you.
Please let us know if you need any help

**Comment by swangi-kumari** [comment]
Hey @smeet07 ,
Are you still working on this issue?
Please let us know if you need any help.

**Comment by smeet07** [comment]
Hi, working on it

**Comment by smeet07** [comment]
Hi @swangi-kumari , can I know the status of this issue, if this still needs to be worked on?

**Comment by swangi-kumari** [comment]
Yes @smeet07 , This issue is still open. If you're willing to work on it, you can take it up.
